android-todo-list/architecture-samples-main/app/src/main/java/com/statsig/todoapp/tasks/TasksScreen.kt

```diff
@@ -97,6 +103,31 @@ fun TasksScreen(
     ) { paddingValues ->
         val uiState by viewModel.uiState.collectAsStateWithLifecycle()
 
+        val bannerDynConf = Statsig.getConfig("warning_banner")
+        val bannerWarningMsg =
+            bannerDynConf.getString("message", "NA") ?: stringResource(R.string.not_applicable)
+        val bannerWarningTextColor =
+            bannerDynConf.getString("textColor", "#ffffff") ?: "#ffffff"
```

Comment:
shouldnt need the ?: defaults - the default value is passed in the `getString` method

Reply:
The `getString` returned a nullable String so added a default value. But I will make it a Nullable as mentioned in the below comment.